### ISSUE TITLE
Regression: opened left bar does not expand anything

### ISSUE THREAD

**Issue by odiferousmint** [vector-im/riot-web#14782]
![2020-07-28-041032_246x211_scrot](https://user-images.githubusercontent.com/43568459/88611047-8133b280-d088-11ea-9ca0-3b778de41c2a.png)

At least there are tooltips now! :P

### Version information

- **OS**: Arch Linux
- **Version**: 1.7.2-1 (element-desktop)

**Comment by odiferousmint** [comment]
Hmm, it's even buggier than I thought. If I un-collapse, collapse, and un-collapse rooms, I get this:

![2020-07-28-041550_248x157_scrot](https://user-images.githubusercontent.com/43568459/88611398-39615b00-d089-11ea-89bf-c34a23e1fb08.png)

**Comment by aWeinzierl** [comment]
Same on Windows (only the effects of the first screenshot though)
Element version: 1.7.2 (automatically provided)

**Comment by Caspy7** [comment]
My experience:
I keep the site (chat.mozilla.org) zoomed to 133%. On Firefox Nightly.
Previously after *each* browser restart the left panel is collapsed and I have to resize it to show the full names as I prefer.
Today Element said it needed to reload to update so I did and got the same result as this issue describes.

I resolved it by setting zoom level to 100% and reloading the page. I then expand the panel again and increased zoom to 133%.
****
Well, great, this solution has been hit or miss. While leaving this comment I've played around and sometimes it works, sometimes it doesn't.
****
So, it seems as though if the panel is not expanded when I hit reload on the page, then the issue reoccurs (no matter the zoom level), but the issue always occurs when zoom is 133% no matter the current page state.
